### ISSUE TITLE
DateFormat is not supported 

### ISSUE THREAD

**Issue by Cellaryllis** [deakjahn/l10n_esperanto#4]
Hey there!

Thanks for creating this package. I did notice an issue when trying to use DateFormat.

We get the following error:
![image](https://user-images.githubusercontent.com/95936504/234073125-47ba284e-209b-4d9c-a095-85d7a5665cfc.png)
![image](https://user-images.githubusercontent.com/95936504/234073164-5a921881-2e28-4961-9d5b-4e9c6d372bd9.png)

eo isn't in the dateTimeSymbols.

We use DateFormat extensively in the package so we would appreciate if it could be supported. We can ask our translators to help translating that if you need help with it.

If there's something we're doing wrong and this should be supported let us know!

**Comment by Cellaryllis** [comment]
Example code if you want to try this out yourself:

```dart
  DateFormat format = DateFormat.MMMd(getLocale().toLanguageTag());
  if (MediaQuery.of(context).alwaysUse24HourFormat) {
    format.add_Hm();
  } else {
    format.add_jm();
  }
```

**Comment by deakjahn** [comment]
Yes, the example helps, I'll look into it. Thanks, there's no need for translators.

**Comment by deakjahn** [comment]
There's an equivalent issue in `intl`'s queue, https://github.com/dart-lang/i18n/issues/88. I'll check out whether it helps and whether this package implemented everything form the Flutter docs mentioned or not.

**Comment by deakjahn** [comment]
It's different, so there's hope. :-)

**Comment by deakjahn** [comment]
@Cellaryllis Could you check out this one? Added the new calls and restructured the files but I'm still waiting for an answer in that issue.

Basically, it looks like some initialization problem now. If I use a

    locale: const Locale('eo'),

in the `MaterialApp`, the system calls the loader of `MaterialLocalizationsEo` (you can put a debug print there temporarily) and the `DateFormat` functions work. Without that, there's no initialization and we get the crash. It might still work in your case, if you only ever ask for the Esperanto date format if the app itself is localized to Esperanto, which sounds logical in the first place, but I don't know for sure yet.
